### ISSUE TITLE
Microsoft Word footnote: NVDA+del doesn't announce distance of cursor from left and upper margin

### ISSUE THREAD

**Issue by amenaide** [nvaccess/nvda#15253]
### Steps to reproduce:
1. Open a Word document.
2. Insert a footnote.
3. In the edit box where you enter the text of the footnote, press NVDA+del to hear the cursor distance from page margins.

### Actual behavior:

Nothing happens, error sound is played.
### Expected behavior:

NVDA announces distance from margins.
### NVDA logs, crash dumps and other attachments:
[Error.log](https://github.com/nvaccess/nvda/files/12250555/Error.log)
#### NVDA installed/portable/running from source:
Portable
#### NVDA version:
2023.2beta1 (2023.2.0.28773)
#### Windows version:
11 Pro, 22H2 (Build 22621.2070)
#### Name and version of other software in use when reproducing the issue:
Microsoft Word, Microsoft 365 subscription
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
